teku/helm/templates/validator-deployment.yaml

```diff
@@ -44,7 +44,7 @@ spec:
           readOnly: false
```

Comment:
should it be `true` too?

Reply:
Teku will write a lock file for each key when the validator client is running so it will need the ability to write.